### PR TITLE
CSC-2131: add id=runsqlscripts; add scripts for ucbg

### PR DESCRIPTION
**Pull request by lkvoong**

### CODE REVIEW

File: services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml [hunk 1]
```diff
@@ -41,18 +41,6 @@
 						<service:key>sqlScriptName</service:key>
 						<service:value>grant_pahma_reporters_privileges.sql</service:value>
 					</service:property>
```
Comment (remillet) [inline services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml:43]: In lines 44-55, you're removing what appears to be three function definitions from PAHMA's tenancy.  Is this really your intent?

**Review by remillet — CHANGES_REQUESTED** [review]

File: services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml (hunk 1 above)
Comment (lkvoong) [reply services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml:43]: Yes, those functions were installed long ago when we made the UOC reports.  There's no reason they need to continue to be created/recreated.  They exist in Prod.

Comment (remillet) [reply services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml:43]: Lam, when I run a "clean" build of the PAHMA sources, I should get ALL functions currently installed in PROD/production.

Comment (lkvoong) [reply services/common/src/main/cspace/config/services/tenants/pahma/pahma-tenant-bindings.delta.xml]: added PAHMA UOC functions back to runsqlscripts; will create new Jira to update runsqlscripts to build deployment-specific objects.

**Pull request merged**